ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -85,6 +95,259 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P6S Polyominoid Tether Collect',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '00CF' }),
+      run: (data, matches) => {
+        data.tileTethers.push(matches);
+      },
+    },
+    {
+      id: 'P6S Polyominoid MapEffect Collect',
+      type: 'MapEffect',
+      netRegex: NetRegexes.mapEffect({ flags: ['00020001', '00400020'] }),
```

Comment:
I dislike these two flags being repeated here and then below, feels like a "magic numbers" situation. I'd rather see a `const crossTile = ...; const plusTile = ...;` at the top of the file and then using those below so it's easier to tell at a glance what these correspond to.